GigyaSwift/Extensions/Notification.Name.swift

```diff
@@ -6,10 +6,10 @@
 //  Copyright © 2019 Gigya. All rights reserved.
 //
 
-import NotificationCenter
-
-extension Notification.Name {
-    static var didGigyaSessionExpire = Notification.Name(rawValue: "didGigyaSessionExpire")
-
-    static var didInvalidateSession = Notification.Name(rawValue: "didInvalidateSession")
-}
+//import NotificationCenter
```

Comment:
I think we can go ahead and delete the code instead of commenting it out

Reply:
I was trying to make it easy to see what modifications were done and for future update to hopefully one day get back to the non-forked, latest version.